homeassistant/components/ovos/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Support for OpenVoiceOS (OVOS) and Neon AI."""
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+DOMAIN = "ovos"
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
New integrations need to use a config flow to be configured. You can create one using `python3 -m script.scaffold`

Reply:
```
❯ python3 -m script.scaffold
usage: __main__.py [-h] [--develop] [--integration INTEGRATION]
                   {significant_change,integration,config_flow,config_flow_oauth2,reproduce_state,device_action,config_flow_discovery,backup,device_condition,config_flow_helper,device_trigger}
__main__.py: error: the following arguments are required: template
```